library/src/main/java/pl/rspective/voucherify/android/view/OnValidatedListener.java

```diff
@@ -0,0 +1,11 @@
+package pl.rspective.voucherify.android.view;
+
+import pl.rspective.voucherify.android.client.model.VoucherResponse;
+import retrofit.RetrofitError;
+
+public interface OnValidatedListener {
+
+    void onValidated(VoucherResponse result);
```

Comment:
Finally I spilt `onValidated` to `onValid` and `onInvalid`. I think it will be more clear.